scripts/utils.ts

```diff
@@ -350,7 +350,7 @@ export async function checkInstanceHealth(instanceId: string, region: string = "
         InstanceIds: [instanceId],
         DocumentName: "AWS-RunShellScript",
         Parameters: {
-          commands: ["screen -ls"]
+          commands: ["cd vargasjr_dev_agent-* && npm run healthcheck"]
```

Comment:
Agent not healthy yet (offline: SSM Command Failed: SSM command failed: /var/lib/amazon/ssm/i-030cd4b345dad40af/document/orchestration/34eb8a39-3822-4e05-8203-aab8c98d118f/awsrunShellScript/0.awsrunShellScript/_script.sh: 1: cd: can't cd to vargasjr_dev_agent-*

We know what the file path will be, based on the version. See ci.yaml for details